### ISSUE TITLE
verifying remote signing / web3signer

### ISSUE THREAD

**Issue by franjoespejo** [prysmaticlabs/prysm#14550]
# 🚀 Feature Request

### Description

Prysm already implements the w3signer/remote-signer standard https://docs.prylabs.network/docs/wallet/web3signer.

The standard assumes the w3s trust the validator client, since there is no way for the w3s to verify the payload to be signed. 

### Describe the solution you'd like

The proposal to fix that is:  https://github.com/ethereum/remote-signing-api/pull/10, 

It is specially useful for diva which distributes the w3s signatures.

### Describe alternatives you've considered

Nimbus has implemented this: https://nimbus.guide/web3signer.html#verifying-web3signer,

**Comment by prestonvanloon** [comment]
Added the "help wanted" label. This seems to be a pretty specific use case for Diva/untrusted setups and we are quite limited on eng bandwidth. If Diva developers or anyone that wants this feature could implement it, then we would be able to allocate limited resources to review of the implementation.

**Comment by DarkLord017** [comment]
@franjoespejo @prestonvanloon can i work on this ?

**Comment by fernantho** [comment]
This issue is a "nice-to-have" in [Jun's](https://github.com/syjn99) and my [EPF Project](https://hackmd.io/@junsong/HkQ9XBEHel).
Once SSZ-QL is ready, this should be very straightforward to implement.
